sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -388,7 +388,7 @@ private[ui] class ExecutionPagedTable(
         +details
       </span> ++
       <div class="stage-details collapsed">
-        <pre>{execution.description}<br></br>{execution.details}</pre>
+        <pre>{Unparsed(execution.description)}<br></br>{execution.details}</pre>
```

Comment:
It makes some sense. My only paranoid worry is that this opens up some type of XSS attack... can a user control the execution description? we'd render the raw HTML. Instead, can we get HTML out of the description? I don't see why it would have that to begin with.

Reply:
@srowen Sorry, I am not very familiar with XSS attack. But IMHO, it is a safe usage here and other places in Spark.

Reply:
I guess the better question is, why is there HTML in the description to begin with? There shouldn't be. The UI is the only component that should add HTML. That would be a better fix, but I don't know where the HTML comes from here.